diff --git a/toolkit/moz.configure b/toolkit/moz.configure
index 07c1fe054dcd938e432b06a5334ec73da01f0c6a..65f9633889b7ae74571ba9ce06ff8118d3160cae 100644
--- a/toolkit/moz.configure
+++ b/toolkit/moz.configure
@@ -26,7 +26,7 @@ project_flag(
 
 project_flag(
     env="MOZ_APP_UA_NAME",
-    default="",
+    default="Firefox",
     nargs=1,
     help="Application name in the User Agent string",
 )
